vendor.conf

```diff
@@ -18,6 +18,9 @@ golang.org/x/sys 07c182904dbd53199946ba614a412c61d3c548f5
 github.com/docker/go-units 9e638d38cf6977a37a8ea0078f3ee75a7cdb2dd1
 github.com/docker/go-connections 3ede32e2033de7505e6500d6c868c2b9ed9f169d
 golang.org/x/text f72d8390a633d5dfb0cc84043294db9f6c935756
+golang.org/x/text/width/tables.go dc401f0b44272155af14de5aa0fd4e0ce8e80851
+golang.org/x/text/width/trieval.go fc7fa097411d30e6708badff276c4c164425590c
```

Comment:
Please remove *.go lines and just update commit hash of `golang.org/x/text`

Reply:
> golang.org/x/text/width/gen.go <commit hash>

This was just my copy-paste error. I meant `golang.org/x/text <commit hash>`.
Sorry for causing confusion.

Reply:
problem still exists. @-@ Do I need to update the whole text package?

Reply:
Vendor check passed after I update text package. :)